src/core/ItemContainer.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from "react";
+import { ComponentCompat } from "../utils/ComponentCompat";
+
+export interface ItemContainerProps {
+    horizontal: boolean;
+    layoutOffset: number;
+    layoutInterval: number;
```

Comment:
change the name

Reply:
The prop names have been changed to scrollOffset and windowSize